client/src/components/KanbanTodo/index.tsx

```diff
@@ -1,62 +1,52 @@
 import React, { useState } from 'react';
 import { useStoryState, useStoryDispatch } from '@/lib/hooks/useContextHooks';
-import { createStory, updateStoryWithName } from '@/lib/api/story';
-import useInput from '@/lib/hooks/useInput';
+import { createStory, deleteStoryWitId } from '@/lib/api/story';
 import Styled from '@/components/KanbanTodo/style';
-import Button from '@/lib/design/Button';
-import Modal from '@/lib/design/Modal';
+import { Button, Modal } from '@/lib/design';
+import { KanbanItem } from '@/components';
 import { StoryType } from '@/types/story';
 
 interface KanbanProps {
   projectId?: number;
 }
 
 const extractLastID = (array: Array<StoryType>) => {
-  if (array.length === 0) return 0;
-  return array.sort((a, b) => b.id - a.id)[0].id;
+  if (!array || array.length === 0) return 0;
```

Comment:
```suggestion
  if (!array?.length) return 0;
```
이렇게하면 더 간결하고 좋을 것 같아요